portal/src/graphql/portal/mutations/updateAppConfigMutation.ts

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useMutation, gql } from "@apollo/client";
+
+import { client } from "../../portal/apollo";
+import { PortalAPIApp } from "../../../types";
+import { UpdateAppConfigMutation } from "./__generated__/UpdateAppConfigMutation";
+
+// relative to project root
+const UPDATE_FILE_PATH = "./authgear.yaml";
```

Comment:
Let's name it more accurately (e.g. `APP_CONFIG_PATH`), since later we would also need to edit `authgear.secrets.yaml`.